pypiserver/core.py

```diff
@@ -246,6 +250,15 @@ def _listdir(root):
                               fn=fn, root=root,
                               relfn=fn[len(root) + 1:])
 
+try:
+    from .cache import listdir_cache
+
+    def listdir(root):
+        return listdir_cache.get(root, _listdir)
+except ImportError:
+    listdir = _listdir
```

Comment:
Yup as I thought. This section was removed in https://github.com/pypiserver/pypiserver/commit/18dc199e18b831f81c15921792a8db5696026341 and because I was hurrying I didn't look closely enough at what git was telling me.